pkg/filter/apiaggregator/apiaggregator.go

```diff
@@ -48,45 +48,56 @@ const (
 var results = []string{resultFailed}
 
 func init() {
-	// FIXME: Rewrite APIAggregator because the HTTPProxy is eliminated
-	// I(@xxx7xxxx) think we should not empower filter to cross pipelines.
-
-	// httppipeline.Register(&APIAggregator{})
+	httppipeline.Register(&APIAggregator{})
 }
 
 type (
-	// APIAggregator is the entity to complete rate limiting.
+	// APIAggregator is a filter to aggregate several HTTP API responses.
 	APIAggregator struct {
 		filterSpec *httppipeline.FilterSpec
 		spec       *Spec
 
-		muxMapper protocol.MuxMapper
+		// rctc is for getting Pipeline in default namespace.
+		rctc *rawconfigtrafficcontroller.RawConfigTrafficController
 	}
 
-	// Spec describes APIAggregator.
+	// Spec is APIAggregator's spec.
 	Spec struct {
 		// MaxBodyBytes in [0, 10MB]
-		MaxBodyBytes   int64  `yaml:"maxBodyBytes" jsonschema:"omitempty,minimum=0,maximum=102400"`
-		PartialSucceed bool   `yaml:"partialSucceed"`
-		Timeout        string `yaml:"timeout" jsonschema:"omitempty,format=duration"`
-		MergeResponse  bool   `yaml:"mergeResponse"`
+		MaxBodyBytes int64 `yaml:"maxBodyBytes" jsonschema:"omitempty,minimum=0,maximum=102400"`
```

Comment:
Prose add `Namespace` in spec with default value `default`, in case of another component use this filter to aggregate apis.

Reply:
If we provide this `namespace` filed, that means I can create a Pipeline with several aggregators to represent different namespace's Pipelines. That's too powerful as just being a filter. (be able to help one pipeline across different namespaces). 
So I prefer to keep this filter be used only in the `default` namespace currently.